doc/source/building/working_with_images/custom_fstab_extension.rst

```diff
@@ -0,0 +1,63 @@
+.. _custom_fstab_extension:
+
+Add or Update the Fstab File
+============================
+
+.. sidebar:: Abstract
+
+   This page provides further information for customizing
+   the `/etc/fstab` file which controls the mounting of
+   filesystems at boot time.
+
+In KIWI, all major filesystems that were created at image
+build time are handled by KIWI itself and setup in `etc/fstab`.
```

Comment:
These should be two `

Reply:
it rendered fine, I have done a testbuild (tox -e doc)  before submission. We use the single backticks in several other places of the doc source.

Thanks for the review looking into the other issues asap

Reply:
Oops, I always thought that one has to use two backticks.